src/main/kotlin/minesweeper/model/Cell.kt

```diff
@@ -1,17 +1,28 @@
 package minesweeper.model
 
-/**
- * @author 이상준
- */
 class Cell(
     val row: Int = 0,
     val column: Int = 0,
 ) {
     private var isMine: Boolean = false
+    private var mineAroundCount: Int = 0
 
     fun isMine(): Boolean = isMine
 
     fun addMine() {
         isMine = true
     }
+
+    fun addMineAroundCount() {
+        mineAroundCount++
+    }
+
+    fun mineAroundCount(): Int = mineAroundCount
```

Comment:
단순 mineAroundCount를 반환하는 것이면 함수가 아니라 변수선언한 mineAroundCount의 getter를 제어하면 되지 않을까요?

```suggestion
    var mineAroundCount: Int = 0
        private set
```

Reply:
넵~